packages/dds/map/src/directory.ts

```diff
@@ -790,9 +790,9 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
                 const result: ISerializableValue = {
                     type: value.type,
                     // eslint-disable-next-line @typescript-eslint/ban-types
-                    value: value.value && JSON.parse(value.value) as object,
```

Comment:
@ChumpChief Can you help me understand what `value` should be set to here? Should it be the result of `JSON.parse(value.value)`? Or should it be a boolean value? My read of the old code is that it should be a boolean, but that doesn't make sense.